packages/design-system/src/components/tooltip/tail.js

```diff
@@ -27,7 +27,7 @@ import { PLACEMENT } from '../popup';
 
 export const SVG_TOOLTIP_TAIL_ID = 'tooltip-tail';
 const TAIL_WIDTH = 34;
-const TAIL_HEIGHT = 8;
+export const TAIL_HEIGHT = 8;
```

Comment:
💭 Probably not worth updating but it seems like there should be two different variables here and they happen to be the same value:
1. The tail height (8px)
2. The tooltip's distance away from the anchor element (also 8px)

In the future it may be beneficial to split these up, if ever the tooltip should be placed a little further/closer to the anchor.

But I don't think it's worth updating now. Maybe it would be helpful to add a comment saying the extra displacement is necessary because the tooltip is always 8px from the anchor?

Reply:
Yeah, Tooltip has the const `SPACE_BETWEEN_TOOLTIP_AND_ELEMENT = 8;` perhaps the cleanest way to update / sync these would be to edit `const SPACE_BETWEEN_TOOLTIP_AND_ELEMENT = TAIL_HEIGHT`?

Reply:
Or vise versa?

Reply:
If the space is dependent specifically on the tooltip height then I'd do something like

```js
const SPACE_BETWEEN_TOOLTIP_AND_ELEMENT = TAIL_HEIGHT + extraOffset;
```

If it isn't I'd do 
```js
const SPACE_BETWEEN_TOOLTIP_AND_ELEMENT = 8;
```

Reply:
I think I got us confused. `const SPACE_BETWEEN_TOOLTIP_AND_ELEMENT = 8;` Already exists on the Tooltip component [here](https://github.com/GoogleForCreators/web-stories-wp/blob/main/packages/design-system/src/components/tooltip/index.js#L42).

Since there is both 
`const SPACE_BETWEEN_TOOLTIP_AND_ELEMENT = 8;`
and 
`const TAIL_HEIGHT = 8;`

I would think we can keep them in sync by updating one to equal the other. I am assuming the `SPACE_BETWEEN_TOOLTIP_AND_ELEMENT` should be the same as the `TAIL_HEIGHT` so that there is enough room for the tail to be placed. 

But now I see just how confusing this is. I _think_ this would be easier to follow if we did something like 
``` 
const TAIL_HEIGHT = 8;
...
const SPACE_BETWEEN_TOOLTIP_AND_ELEMENT = TAIL_HEIGHT;
...
//  In order to check if there's an overlap with the window's bottom edge we need the overall height of the tooltip
//  from the anchor's y position along with the amount of space between the anchor and the tooltip content. 
 const neededVerticalSpace = offset.y + offset.popupHeight + SPACE_BETWEEN_TOOLTIP_AND_ELEMENT;
```